CGPs/template.md

```diff
@@ -0,0 +1,41 @@
+# CIP [0000]: Title for Governance Proposal
+
+- Date: 2020-05-11
+- Author: @githubhandle
+- Status: DRAFT|PROPOSED|EXECUTED|EXPIRED
+	- DRAFT = Feedback collection
+	- PROPOSED = The governance proposal has been submitted to the network
+	- EXECUTED = The governance proposal was executed on the network
+	- EXPIRED = The governance proposal expired
+- Governance Propsal ID #: [if submitted]
+- Date Executed: [if executed]
+
+## Overview
+
+Pick a four digit number subsequent to the latest PR raised and name your file that.
```

Comment:
This may be hard to understand for non native english speakers. Maybe put an example?

Reply:
consistency with the CIP instruction -- ya its kinda confusing for me even.  I'll add example